src/tests/stub_errorpage.cc

```diff
@@ -7,17 +7,31 @@
  */
 
 #include "squid.h"
-#include "errorpage.h"
+#include "AccessLogEntry.h"
+#include "HttpReply.h"
 
 #define STUB_API "errorpage.cc"
 #include "tests/STUB.h"
 
-err_type errorReservePageId(const char *, const SBuf &) STUB_RETVAL(err_type(0))
+#include "errorpage.h"
+CBDATA_CLASS_INIT(ErrorState);
+ErrorState::ErrorState(err_type, Http::StatusCode, HttpRequest *, const AccessLogEntryPointer &) STUB
+ErrorState::ErrorState(HttpRequest *, HttpReply *, const AccessLogEntryPointer &) STUB
+ErrorState::~ErrorState() STUB
+ErrorState *ErrorState::NewForwarding(err_type, HttpRequestPointer &, const AccessLogEntryPointer &) STUB_RETVAL(nullptr)
+HttpReply *ErrorState::BuildHttpReply(void) STUB_RETVAL(nullptr)
+void ErrorState::validate() STUB
+void errorInitialize(void) STUB
+void errorClean(void) STUB
+void errorSend(const Comm::ConnectionPointer &, ErrorState *) STUB
 void errorAppendEntry(StoreEntry *, ErrorState * ) STUB
-bool strHdrAcptLangGetItem(const String &, char *, int, size_t &) STUB_RETVAL(false)
-void TemplateFile::loadDefault() STUB
+err_type errorReservePageId(const char *, const SBuf &) STUB_RETVAL(err_type(0))
```

Comment:
Why move these three unchanged definitions? If possible, please keep their positions unchanged (from diff point of view).

Reply:
Side effect of updating the file to the latest stub layout style. Each line in the stub is being kept in the same order as the matching .h it came from relative to the other lines - which in turn enables semi-automated stub creation.

Reply:
> Each line in the stub is being kept in the same order as the matching .h it came from relative to the other lines

I object to introducing that kind of requirement (for humans) until we automate stub maintenance.

Reply:
It is not a requirement. It is how I generate these stubs. This file was completely re-generated so its contents will not line up exactly with the ad-hoc human content that was previously there.

Reply:
> Alex: Why move these three unchanged definitions? If possible, please keep their positions unchanged (from diff point of view).

> Amos: Side effect of updating the file to the latest stub layout style. Each line in the stub is being kept in the same order as the matching .h it came from relative to the other lines

> Alex: I object to introducing that kind of requirement (for humans) until we automate stub maintenance.

> Amos: It is not a requirement. It is how I generate these stubs.

Aha, so there is no relevant "latest stub layout style" as far as Project is concerned. There is some generator program that introduces unwanted changes into PRs like this one. A developer can use whatever tools they want, of course, but I think they should clean up after their tools. This change request is asking for such a cleanup: Please do not reorder unchanged stubs in PRs (even if your generator reorders unchanged stubs).

Reply:
I've restored the ordering. TBH I find these concerns do not meet the bar for relevance, but
that goes both ways so I've restored the previous state